doc/reference/reference_lua/box_space/create_index.rst

```diff
@@ -305,16 +305,18 @@ Allowing null for an indexed key
 
 If the index type is TREE, and the index is not the primary index,
 then the ``parts={...}`` clause may include ``is_nullable=true`` or
-``is_nullable=false`` (the default). If ``is_nullable`` is true,
-then it is legal to insert ``nil`` or an equivalent such as ``msgpack.NULL``.
+``is_nullable=false`` (the default).
+
+If ``is_nullable`` is true, then it is legal to insert ``nil`` or an equivalent
+such as ``msgpack.NULL``.
 It is also legal to insert nothing at all when using trailing nullable fields.
 Within indexes, such "null values" are always treated as equal to other null
```

Comment:
Maybe `"null values"` -> `"null" values`?

Reply:
Or just null values, without quotes. We don't use quotes in the next three usages in the same paragraph.